src/ext/resources/legacygraphqlapi/schema.graphqls

```diff
@@ -1448,11 +1538,18 @@ type Money {
     amount: Float!
 }
 
+""""
+An ISO-8601-formatted datetime with offset, i.e. `2023-06-13T14:30+03:00` for 2:30pm on June 13th 2023 at Helsinki's offset from UTC at that time.
+"""
+scalar DateTimeWithOffset
```

Comment:
In java this is called `OffsetDateTime`. Do we want to use this name?

Reply:
I thought about it but OffsetDateTime could be understood to mean a datetime that represents an offset (I know that doesn't make much sense). Although, offset can also mean that something has been "offsetted".